transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -346,6 +346,61 @@ static jint netty_epoll_native_sendmmsg0(JNIEnv* env, jclass clazz, jint fd, jbo
     return (jint) res;
 }
 
+static jint netty_epoll_native_recvmmsg0(JNIEnv* env, jclass clazz, jint fd, jboolean ipv6, jobjectArray packets, jint offset, jint len) {
+    struct mmsghdr msg[len];
+    memset(msg, 0, sizeof(msg));
+    struct sockaddr_storage addr[len];
```

Comment:
Shouldn't the size of this struct be calculated based on whether the message being handled is ipv4 or ipv6?

Reply:
sockaddr_storage can hold either... so not sure what I am missing ?

Reply:
Seems odd that len is used to init msg and addr, my point was it seems more logical to init sockaddr_storage with a fixed size based on the protocol type, what if the length of the message received is smaller than the required size to init sockaddr_storage?

Reply:
Len is the number of max messages... not the size of the struct itself.

Reply:
Hah ok that makes sense, based on the method signature it looked like the length of the payload.